protocol/port-forwarding.md

```diff
@@ -11,6 +11,10 @@ keywords:
     - Subspace Desktop
 ---
 
+:::caution
+If you are using Linux, verify you have opened up your firewall for TCP traffic on Port 30333 with the following command `sudo ufw allow 30333/tcp`
+:::
```

Comment:
This is only applicable if they enabled `ufw`, which is not the case by default. Also it is probably Ubuntu-specific, not sure it is the case on all Linux distros.